internal/controller/account_resolver.go

```diff
@@ -101,6 +101,7 @@ func (bar *BOPAccountIdResolver) MapClientIdToAccountId(ctx context.Context, cli
 	}
 
 	logger.WithFields(logrus.Fields{"account": jsonData.Identity.AccountNumber}).Debug("Located account number for client")
+	logger.WithFields(logrus.Fields{"org_id": jsonData.Identity.Internal.OrgID}).Debug("Located OrgID for client")
```

Comment:
I think we should combine this new log message (which contains the org_id) with the log message above (line 103) and just write a single log message which contains both the account number and org_id.